Created app for user authentication

HJLee1130 opened the pull request:
created **account** app for user login, logout, and registration.
I used crispy form (a third party Django app). Install using pip:
pip install --upgrade django-crispy-forms

HJLee1130 committed:
Add files via upload

gschua reviewed (changes requested):
- Next time, we should put all our commits in one pull request, else GitHub will go nuts when we try to merge.
 - 1st branch: No "Confirm Password" field?
 - 2nd branch: closed, commited the updated setting.py to this branch to avoid merging issues.
 - 3rd branch: **Not** supposed to create urls.py or views.py in *vapoursite*. Please update the urls.py and views.py in *fume* with your changes instead. I've closed this branch also, so after you update, commit the changes to this branch.
 - 4th branch: don't update the main*.html files because the ones on the master branch are outdated. I'll link them to your app, don't worry. I'll close this branch.

gschua committed:
By HyunJi: updated vapoursite/settings.py

By HyunJi: enable crispy form by adding "crispy_forms"under INSTALLED_APPS in vapoursite/settings.py
